openapi.yaml

```diff
@@ -720,3 +837,123 @@ components:
           title: Current page number
           type: string
           example: "1"
+    AclRequest:
+      required:
+        - aclPatternType
+        - topicname
+        - environment
+        - teamname
+        - topictype
+        - aclIpPrincipleType
+      type: object
+      properties:
+        remarks:
+          pattern: ^$|^[a-zA-Z 0-9_.,-]{3,}$
+          type: string
+          example: "Hello, thank you."
+          description: "A comment on the request for the approver."
+        consumergroup:
+          pattern: ^$|^[a-zA-Z0-9_.-]{3,}$
+          type: string
+          example: "Group-one"
+          description: "This is mandatory if topictype is consumer"
+        acl_ip:
+          type: array
+          items:
+            type: string
+          example: ["35.239.43.144"]
+        acl_ssl:
+          type: array
+          items:
+            type: string
+          example: ["username", "username-two"]
+        aclPatternType:
+          type: string
+          enum: ["LITERAL", "PREFIXED"]
+          example: "LITERAL"
+          description: "If topictype is consumer, can only be LITERAL. If topictype is producer, can be LITERAL or PREFIXED"
+        transactionalId:
+          pattern: ^$|^[a-zA-Z0-9_.-]{3,}$
+          type: string
+          example: "id-123"
+          description: "Only available if aclPatternType is LITERAL"
+        req_no:
+          type: integer
+          format: int32
+          example: 100
+        topicname:
+          type: string
+          example: "myTopic"
+          description: "Only topics available in chosen environment are allowed"
+        environment:
+          type: string
+          example: "1"
+          description: "ID of environment"
+        environmentName:
+          type: string
+          example: "DEV"
+          description: "Name of environment"
+        teamname:
+          type: string
+          example: "Ospo"
+        teamId:
+          type: integer
+          format: int32
+          example: 1
+        requestingteam:
+          type: integer
+          format: int32
+          example: 1
+        appname:
+          type: string
+          example: "App"
+        topictype:
+          type: string
+          enum: ["Producer", "Consumer"]
+          example: "Producer"
+        username:
+          type: string
+          example: "User"
+        requesttime:
+          type: string
+          format: date-time
+          example: "2018-03-20T09:12:28Z"
+        requesttimestring:
+          type: string
+          pattern: "dd-MMM-yyyy HH:mm:ss"
+          example: "10-11-2020 10:45:30"
+        aclstatus:
+          type: string
+          enum: ["created", "approved", "denied"]
```

Comment:
```suggestion
          enum: ["created", "approved", "denied", "deleted"]
```

Reply:
Nice, thank you! I thought I might be missing some ^^